### ISSUE TITLE
bitlaunch.io is a SCAM project

### ISSUE THREAD

**Issue by Rapid0nion** [BitLaunchIO/blcli#4]
bitlaunch.io is a SCAM project

After I reloaded my balance they nlock my account without a reason. 

**Avoid this "service"**

**Comment by deadphoenix8091** [comment]
Agree, lost access to domains purchased for year runtimes. I let the account idle at 2€ remaining funds, just got banned after a while now lost access to the domain.

**Comment by wanmei002** [comment]
i switch to [this provider](https://console.enigmhaven.com/login/?invitationCode=dXNlci1DamJBa1I=), if you want to change service provider. click it.
